server/planning/events/events_files.py

```diff
@@ -45,7 +46,13 @@ class EventsFilesService(superdesk.Service):
     def on_create(self, docs):
         for doc in docs:
             # save the media id to retrieve the file later
-            doc['filemeta'] = {'media_id': doc['media']}
+            _file = app.media.get(doc['media'])
+            doc['filemeta'] = {
+                'media_id': doc['media'],
```

Comment:
personally would drop this one too, it's there already as media

Reply:
I thought about it, but not sure, it might break something. Let's try to drop it.